package.json

```diff
@@ -729,6 +729,11 @@
                     "description": "Points to the julia executable.",
                     "scope": "machine-overridable"
                 },
+                "julia.juliaupCommand": {
+                    "type": "string",
+                    "default": "juliaup",
+                    "scope": "machine"
```

Comment:
Can you add a description here? In particular, does this setting allow for version specifiers like `julia +1.6.4`?

Reply:
No, this was the path to the `juliaup.exe` binary. I actually removed it now, thinking more about it that is actually not something that should be configurable.

To use a different Julia, it should (in theory) be enough to just put `julia +1.6.4` into the standard exe path setting.

Reply:
Ah, makes sense.